src/Storefront/Resources/views/storefront/element/cms-element-product-description-reviews.html.twig

```diff
@@ -1,4 +1,4 @@
-{% block element_product_desciption_reviews %}
```

Comment:
This is a breaking change, as plugins that may override this block now need to override the new block.
Therefore you can not simply rename the block, instead you should add a 
```
{# @deprecated tag:v6.5.0 - block will be removed use `element_product_description_reviews` instead #}
```
and add the new block directly inside the old one

Reply:
also add each block you deprecate to the changelog:
```
* Deprecated block `old-one`, use `new-one` instead
``` 
and please also add 
```
___
# Next Major Version Changes
## Renamed product description blocks to fix typo
The following blocks in the `cms-element-product-description-reviews.html.twig` template:
* `old-block` -> `new-block`
```

That way the info get's automatically added to the `UPGRADE_6.5.md` file, and external developers are aware of the deprecation and breaks.